misago/parser/tests/test_parser.py

```diff
@@ -117,3 +117,37 @@ def test_parser_parses_line_breaks_surrounded_by_spaces():
             ],
         },
     ]
+
+def test_parser_parses_crlf_two_paragraphs():
+    result = parse("First paragraph.\r\n\r\nSecond paragraph.")
+    assert result == [
+        {
+            "type": "paragraph",
+            "children": [
+                {"type": "text", "text": "First paragraph."},
+            ],
+        },
+        {
+            "type": "paragraph",
+            "children": [
+                {"type": "text", "text": "Second paragraph."},
+            ],
+        },
+    ]
+
+def test_parser_parses_crlf_skips_extra_linebreaks_between_blocks():
```

Comment:
Can we also get a test where text is split by single `\r\n` that will result in a linebreak AST to appear in paragraph?

Reply:
done